DependencyInjection/HttplugExtension.php

```diff
@@ -203,6 +203,13 @@ private function configurePluginByName($name, Definition $definition, array $con
                 $definition->replaceArgument(1, [
                     'replace' => $config['replace'],
                 ]);
```

Comment:
missing break

Reply:
you still need to add a `break;` here, its the reason why the tests fail

Reply:
@dbu I  fixed all review, can you please review my PR?